src/util/Constants.js

```diff
@@ -42,34 +42,33 @@ function makeImageUrl(root, { format = 'webp', size } = {}) {
 exports.Endpoints = {
   CDN(root) {
     return {
-      Emoji: (emojiId, format = 'webp') => `${root}/emojis/${emojiId}.${format}`,
```

Comment:
I think these should stay in the event anyone manually uses the CDN export..at least for those that actually don't use the function in question (so from the looks of things..this one)

Reply:
Oops didn't realize this one didn't use it, will revert

Reply:
Actually idk why this endpoint doesn't use the makeImageUrl because it should support different sizes, but that would be a semver major so I'll simply revert this change and leave that for v14